components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/OpenAPICodegenUtils.java

```diff
@@ -684,7 +684,8 @@ private static void validateInterceptorAvailability(String interceptorName, bool
             if (path != null && operation != null) {
                 errorMsg += "under path:'" + path + "' operation:'" + operation + "' ";
             }
-            errorMsg += "is not available in the " + GatewayCliConstants.PROJECT_INTERCEPTORS_DIR + " directory.";
+            errorMsg += "is not available in any function in the " + GatewayCliConstants.PROJECT_INTERCEPTORS_DIR +
```

Comment:
errorMsg += "function is not available in the " + GatewayCliConstants.PROJECT_INTERCEPTORS_DIR +

Since it is referring to a function name. not something inside a function.

Reply:
hmm yeah, it should be like
"The interceptor 'validateRequest' mentioned in openAPI definition:'/Users/hasunie/RD/MC/tst/intercept/api_definitions/openapi.yaml' under path:'/store/order/{orderId}' operation:'get' is not available as a function name in interceptors directory"

Reply:
It would be better.